### DISCUSSION TITLE
To effectively prevent duplicate issues, need an option that suggests the existing issues while typing the issue title

### DISCUSSION THREAD

**Discussion by egvijayanand** [community/community#18185] [Projects and Issues]
This has been the case as issues are created and some of them get marked as duplicates. Even though a search is performed prior to creation, it all depends on the search keywords.

This is a request for such a feature to suggest the list of possible matching issues from the issue title by performing a **contextual search**.

Similar ones are available with Windows and Visual Studio feedback, it helps to save time in providing a little or more info on the same issue and to check for the resolution if resolved or to join the thread if not resolved to know the resolution when one made available. If none match, it's gonna be a new issue anyway.

**Comment by egvijayanand** [comment]
It could be based on the combination of platform, modules, supported controls, and any such context that helps to track the issue. Some metadata can be provided by the repository maintainers (owner/collaborator).

Probably an AI model that learns by itself based on the nature of issues.

**Comment by corneliusroemer** [comment]
I wrote a detailed feature proposal here (ironically I only noticed this issue here exists after I wrote mine, case in point!): https://github.com/orgs/community/discussions/137345

Another related discussion is: https://github.com/orgs/community/discussions/8277

Gitlab does it like that:
![2024-08-31 16 40 03](https://github.com/user-attachments/assets/f7834f6b-ffdf-4ac0-857c-b7c449d997a4)
